docs/docs/orm-support.md

```diff
@@ -25,4 +29,38 @@ const db = drizzle(client);
 await db.select().from(...);
 ```
 
-See the [Drizzle documentation](https://orm.drizzle.team/docs/get-started-postgresql#pglite) for more details.
+See the [Drizzle documentation](https://orm.drizzle.team/docs/get-started-postgresql#pglite)
+for more details.
+
+## Knex.js
+
+[Knex](https://knexjs.org/) is a stable, reliable Query Builder for various
+database engines. Key features include:
+
+- Query builder
+- Schema builder
+- Raw queries
+- Database migration tool
+
+To use Knex.js with PGlite, add the following dependencies to your project:
```

Comment:
Could you add a note that the `knex-pglite` package is unofficial (not affiliated with knex)